lib/stanford-mods/vocabularies/searchworks_languages.rb

```diff
@@ -364,7 +364,7 @@
   'pan' => 'Panjabi',
   'pap' => 'Papiamento',
   'pau' => 'Palauan',
-  'peo' => 'Old Persian (ca. 600-400 B.C.)',
+  'peo' => 'Old Persian (ca. 600-400 BCE)',
```

Comment:
I don't think we should change the controlled vocabulary term as part of this change. If nothing else, it will need more coordination to do.

Reply:
Oops, I think was a little too quick with a find-replace here